src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = sourceString.split(';');
+
+  const finalResult = {};
+
+  result.forEach((element) => {
+    const [key, value] = element.split(':');
```

Comment:
[CODE KNOWLEDGE] - Check if the `split` operation could result in more than two items in the array. If it's possible, handle cases with multiple colons within the style string's property values, even though it's not required for this task. If not, ignore this comment.